docs/pages/versions/unversioned/sdk/securestore.mdx

```diff
@@ -107,6 +114,39 @@ When using `expo-secure-store`, you can set the [`ios.config.usesNonExemptEncryp
 
 Setting this property automatically handles the compliance information prompt.
 
+## Android Auto Backup
+
+Android Auto Backup for Apps automatically backs up a user's data from apps that target and run on Android 6.0 (API level 23) or higher.
+The Auto Backup system has to be configured to exclude `expo-secure-store` data from the backup, as it's impossible to decrypt it after
+restoring the backup - the app will be using a different KeyStore.
```

Comment:
```suggestion
The Auto Backup system has to be configured to exclude `expo-secure-store` data from the backup since it's impossible to decrypt it after restoring the backup &mdash; the app will use a different Keystore.
```

Will the app use a different Keystore after restoring the backup or the app requires the keystore to be generated once again? Asking because I found the last part of the sentence confusing.

Reply:
I changed the wording a bit, as there is only one Android KeyStore and just the app's entries are deleted 😅 
Entry creation is handled by the module when the user tries to save data